HerPublicWebsite.UnitTests/BusinessLogic/Services/DateHelperTests.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using HerPublicWebsite.BusinessLogic.Services.RegularJobs;
+using NUnit.Framework;
+
+namespace Tests.BusinessLogic.Services;
+
+public class DateHelperTests
+{
+    [Test]
+    public void StartOfMonthService_WhenCalledWithoutParameter_ReturnsAPreviousDate()
+    {
+        // arrange
+        var startOfMonthService = new DateHelper();
+        
+        // act
+        var result = startOfMonthService.GetStartOfPreviousMonth();
+        
+        // assert
+        Assert.That(result <= DateTime.Today);
```

Comment:
You should not be using `Assert.That(…)` for assertions.

The project uses [FluentAssertions](https://fluentassertions.com/) for readability and better failure messages. Please rewrite using those.